sirmordred/task_identities.py

```diff
@@ -50,6 +51,30 @@
 logger = logging.getLogger(__name__)
 
 
+def get_file_hash(filepath):
+    """Generate hash based on the file content
+
+    Read the content of a JSON file, remove all no alphanumeric
+    characters, sort the remaining characters and create a hash from them.
+
+    :param filepath: local path of the file to hash
+    :return: content hash
+    """
+    with open(filepath, 'r') as f:
+        content = f.read()
+
+    json_content = json.loads(content)
+    # Remove the attribute time for identities file, since its value changes
+    # any time the file is generated
+    json_content.pop('time', None)
```

Comment:
Any reason for this? The idea is the time has changed probably it's a different file. Is there any reason why the file should be the same but with a different timestamp?

Reply:
The piece of at https://github.com/chaoss/grimoirelab-sirmordred/blob/master/sirmordred/task_identities.py#L226 is called to convert the yaml file to JSON at any loop. The JSON file generated contains the `time` attribute, which makes the hash comparison fail.

The modification has been mainly done on the method `load_identities_file` to avoid modifying too much the code. However, if you think it's worthy I can see how to change the code.